### PR TITLE
Bug smashing

### PR DESCRIPTION
**Pull request by brianlovin**
**Status**
- [x] WIP
- [ ] Ready for review
- [ ] Needs testing

**Deploy after merge (delete what needn't be deployed)**
- api
- hyperion (frontend)
- athena

---

- https://github.com/withspectrum/spectrum/pull/4072/commits/1f16c0c04ac78b45f63eb376dec1d0a0bc0aca4b fixes https://sentry.io/space-program/spectrum/issues/576605081/?query=is:unresolved
- https://github.com/withspectrum/spectrum/pull/4072/commits/ba2d5a2a593d8708e450fa6f7fd0fb21e165ac5d should fix issues like https://sentry.io/space-program/spectrum/issues/616374538/?query=is:unresolved https://sentry.io/space-program/spectrum/issues/617530648/?query=is:unresolved
- https://github.com/withspectrum/spectrum/pull/4072/commits/0ad50c006993ea78c414c9697670398067a97378 should fix https://sentry.io/space-program/spectrum/issues/714565176/?query=is:unresolved
- https://github.com/withspectrum/spectrum/pull/4072/commits/ff3868be7ffc0b6861c4e30f65cbed2999a64bd8 should fix https://sentry.io/space-program/spectrum/issues/699750881/?query=is:unresolved
- https://github.com/withspectrum/spectrum/pull/4072/commits/f08340679a8a1282976dfff57a99c7e3cd9083fb might fix a safari issue causing https://sentry.io/space-program/spectrum/issues/611105416/?query=is:unresolved
- https://github.com/withspectrum/spectrum/pull/4072/commits/2ac439d080848f3a09b51a6749b583998342f869 cleans up the super gnarly errors being logged in api console
- https://github.com/withspectrum/spectrum/pull/4072/commits/f7d5b0ed64495cf6414b0ffbdcf46a17f3df1f01 needs attention @mxstbr: I found several users without usersSettings records in the db https://sentry.io/space-program/spectrum/issues/658808688/tags/user/ - I don't know why this is, but this commit adds a fallback to populate a default record if none is found. This will fix issues like https://sentry.io/space-program/spectrum/issues/720222992/events/33640089554/
- https://github.com/withspectrum/spectrum/pull/4072/commits/59f2f55ff5256988786a33a031d3cad13c90fe8f fixes the image gallery not working. Have I mentioned how much I hate the fact that the number '0' is falsey?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,6 +83,7 @@
     "draft-js-drag-n-drop-plugin": "^2.0.3",
     "draft-js-embed-plugin": "^1.2.0",
     "draft-js-focus-plugin": "^2.2.0",
+    "draft-js-fork-mxstbr": "npm:draft-js-fork-mxstbr",
```
Comment (brianlovin) [inline package.json]: @mxstbr this was inserted when I ran `yarn upgrade` - any idea why?

**Comment by brianlovin** [comment]
Hm tests are failing after I upgraded some dependencies: "localStorage is not available for opaque origins" super weird @mxstbr, will keep investigating

**Review by mxstbr — APPROVED** [review]
Looks great so far!

**Comment by brianlovin** [comment]
Going to merge this while I focus on a couple other features for the day, so don't want this to get backlogged :)

**Pull request merged**